gensrc/thrift/MVMaintenance.thrift

```diff
@@ -0,0 +1,74 @@
+// This file is made available under Elastic License 2.0
+
+namespace cpp starrocks
+namespace java com.starrocks.thrift
+
+include "Status.thrift"
+include "Types.thrift"
+include "AgentService.thrift"
+include "InternalService.thrift"
+
+struct TMVMaintenanceStartTask {
+    1: required string db_name
+    2: required string mv_name
+    3: optional InternalService.TExecPlanFragmentParams plan_params
+}
+
+struct TMVMaintenanceStopTask {
+}
+
+struct TBinlogLSN {
```

Comment:
How about `TBinlogOffset`? LSN is just a way to tell the offset.

Reply:
All right.